.changeset/swift-gifts-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
+---
+
+Upload and load patch changes ([#748](https://github.com/finos/legend-studio/issues/748)).
```

Comment:
This is a feature, so it's `minor`, also, this is not a bug, so we don't need to link issue. And I think the message should be `Support upload and load patch`.

Reply:
could be a bug since we had download patch but no load patch. but i get your point 👍